src/lib/fhirValidator.ts

```diff
@@ -212,8 +213,14 @@ const fhirValidator = {
       );
     }
 
-    const validator = await this.getValidator(resourceType);
+    if (get(resource, 'contained[0].contained[0]') !== undefined) {
```

Comment:
So, I was not precise enough in my answer to your slack query: if there are multiple contained resources, we need to check that _none_ of them have contained resource. That is, an error should be thrown if `(resource.contained !== undefined) && resource.contained.any((res) => (res.contained !== undefined))` is true.